helm/cert-operator-chart/templates/configmap.yaml

```diff
@@ -16,6 +16,10 @@ data:
           caFile: ''
           crtFile: ''
           keyFile: ''
+      resource:
+        vaultCrt:
+          expirationThreshold: '24h'
+          namespace: 'default'
```

Comment:
The new settings are properly set in the config map. Note that the 24 hours is a magic number I just made up. IMO it should be good enough to let the cert-operator know 24 hours before the certificates expire that it should actually renew them in the Kubernetes secrets.